feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -562,48 +677,77 @@ func TestQoSOutputIntfConfig(t *testing.T) {
 		queueName  string
```

Comment:
rename this field to targetGroup and rename queue to queueName, for consistency with other test cases

Reply:
queueName  string in this line is from the original script.
queueName and queue are the appropriate variable names for the data types in that function.